src/components/common/Actions.vue

```diff
@@ -199,10 +199,7 @@ export default class Actions extends Mixins(DateMixin, FilingTemplateMixin, Lega
       if (this.isBusySaving) return
       this.setIsFilingPaying(true)
 
-      if (
-        this.getIncorporationDateTime.effectiveDate &&
-        !this.isValidDateTime(this.getIncorporationDateTime.effectiveDate)
-      ) {
+      if (!this.isValidDateTime(this.getIncorporationDateTime.effectiveDate)) {
```

Comment:
`isValidDateTime()` already handles empty date, so extra check here was not needed.